composer.json

```diff
@@ -37,17 +37,17 @@
     ],
 
     "require": {
-        "php": ">=5.5",
-        "league/climate": "^2.0|^3.0",
+        "php": "^5.6 || ^7.0",
```

Comment:
This version constraint is weird. It's the same as:

```
^5.6|^7.0
```

Not sure why the double pipe is used.